edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/pvc/KubernetesPvcMapper.cs

```diff
@@ -41,9 +41,9 @@ bool ShouldCreatePvc(Mount mount)
             return this.storageClassName.HasValue || this.persistentVolumeName.HasValue;
         }
 
-        V1PersistentVolumeClaim ExtractPvc(Mount mount, IDictionary<string, string> labels)
+        V1PersistentVolumeClaim ExtractPvc(KubernetesModule module, Mount mount, IDictionary<string, string> labels)
         {
-            string name = KubeUtils.SanitizeK8sValue(mount.Source);
+            string name = KubernetesModule.PvcName(module, mount);
```

Comment:
Nit readability: I would recommend changing variable name because name is super overloaded
```suggestion
            string pvcName = KubernetesModule.PvcName(module, mount);
```

Reply:
Sounds good.

Reply:
I cannot see this change in the file at the moment. Hope you can accept suggestion prior to merge.